contracts/HybridCustody.cdc

```diff
@@ -1073,38 +1100,38 @@ pub contract HybridCustody {
 
         /// Retrieves a reference to the ChildAccount associated with the given parent account if one exists.
         ///
-        pub fun borrowChildAccount(parent: Address): &ChildAccount? {
+        access(Owner) fun borrowChildAccount(parent: Address): auth(Capabilities) &ChildAccount? {
```

Comment:
I didn't realize we could use the built in `Capabilities` entitlement typically used on `Account` - maybe this would be a good entitlement for `GetterPrivate` over `Get`?

Reply:
Ah, yeah this is a lapse, I'll do another pass-through and make sure these are all good